docs/platform/concepts/free-trial.rst

```diff
@@ -0,0 +1,34 @@
+30-day trials
+==============
+
+Aiven offers a free trial for 30 days to explore the platform and the `Aiven services <https://docs.aiven.io/>`_. You don't need a credit card to sign up.
+
+If you create services on the :doc:`free plan </docs/platform/concepts/free-plan>`, they will continue to run alongside your free 30-day trial without affecting your trial credits. Free plan services also continue running after the trial has expired.
+
+Trial features and limitations
+-------------------------------
+
+The 30-day trial gives you full access to the `Aiven Console <https://console.aiven.io/>`_, :doc:`API </docs/tools/api>`, :doc:`CLI </docs/tools/cli>`, :doc:`Aiven Terraform provider </docs/tools/terraform>`, and :doc:`Aiven Operator for Kubernetes </docs/tools/kubernetes>`. The credits can be used on any service types and plan sizes. 
```

Comment:
You also have full access to the console etc without the 30 day trial. The key is you get credits to create non-free services, plus the ability to interact with functionality that requires non-free services (such as creating and placing services into VPCs).